src/elements/hoverboard-analytics.html

```diff
@@ -19,9 +19,9 @@
       (document,'script','../../node_modules/autotrack/autotrack.js');
 
       ga('create', {
-        trackingId: window.ENV === 'prod' ? '{$ analytics.prod $}' : '{$ analytics.dev $}',
```

Comment:
Looks like we don't need this line anymore: https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/index.html#L76

Reply:
yes, I removed it, I might forgot to commit it